### PR TITLE
Email whitelisting feature

### PR DESCRIPTION
**Pull request by seromenho**

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -21,6 +21,20 @@
...
+    <div class="form-group clearfix">
+      <div class="col-sm-12">
+        <div class="checkbox checkbox-icon">
+          <input type="checkbox" id="email-domain">
+          <label for="email-domain">
+            <span class="check"><i class="fa fa-check"></i></span> Enable domain white listing
+          </label>
+        </div>
+      </div>
+      <div class="col-sm-12 email-domains-input hidden">
```
Comment (tonytlwu) [inline interface.html]: This should be wrapped around `<div class="form-group"></div>` the you won't need `.email-domains-input { margin-top: 10px; }` in `interface.css`.

File: js/interface.js [hunk 2]
```diff
@@ -40,87 +40,67 @@ function selectDataSource(ds) {
...
+          key: 'emailMatch',
+          label: 'Select the column with the email address',
+          type: 'single'
+        }]
+      },
+      result: data.dataSourceQuery
+    };
+    break;
+
+  default:
```
Comment (tonytlwu) [inline js/interface.js]: What would happen in this case? Will it continue to pass an empty object `{}` to the DS Query provider? Will this create an error? Maybe the error should either be handled, or `default` to the same case as `email`?

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: interface.html (hunk 1 above)
Comment (seromenho) [reply interface.html]: @hcarneiro ^^ would you mind?

File: js/interface.js (hunk 2 above)
Comment (seromenho) [reply js/interface.js]: Good point. email/default are now the same.

**Review by hcarneiro — APPROVED** [review]

**Pull request merged**